app/src/main.ts

```diff
@@ -159,6 +169,19 @@ if (shouldQuit) {
         });
       });
     }
+    if (!appArgs.disableOldBuildWarning) {
+      if (
+        new Date().getTime() - appArgs.buildDate >
+        OLD_BUILD_WARNING_THRESHOLD_MS
+      ) {
+        void dialog.showMessageBox(null, {
```

Comment:
Why the `void`?

Reply:
Promises must be handled appropriately or explicitly marked as ignored with the `void` [operator.eslint@typescript-eslint/no-floating-promises
](https://github.com/typescript-eslint/typescript-eslint/blob/v4.8.2/packages/eslint-plugin/docs/rules/no-floating-promises.md)

Reply:
Okay, then I'd rather have the ignore, for maximal explicitness:

```suggestion
        // eslint-disable-next-line @typescript-eslint/no-floating-promises
        dialog.showMessageBox(null, {
```